CHANGELOG.md

```diff
@@ -19,7 +19,11 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 + Only orbit file is used (even if multiple specified) to minimize errors and ensure consistency over region
 + GUNW packaging is restructed to store SLC (ref and sec) wet and tropo delays rather than the differential
 
-## [0.4.1]
+##[0.4.2]
```

Comment:
This release already exist on dev, but is being captured as a change. Can you double check the rebasing is done correctly?